spec/double_entry/reporting_spec.rb

```diff
@@ -201,6 +201,34 @@
         end
       end
 
+      context 'filtering by key with multiple values' do
+        let(:code) { :spend }
+        let(:filter) do
+          [
+            :metadata => {
+              :category => category,
```

Comment:
Hi @spickermann! That indeed would be a useful feature. I don't know how to resolve the and/or problem at this point. We don't yet have a use case for it here at Envato so I'm going to play the 'out of scope' card this time. 😀